src/libraries/System.Private.CoreLib/src/System/Decimal.cs

```diff
@@ -243,10 +243,18 @@ public static long ToOACurrency(decimal value)
         // The possible binary representations of a particular value are all
         // equally valid, and all are numerically equivalent.
         //
-        public Decimal(int[] bits)
+        public Decimal(int[] bits) :
+            this(bits != null ? bits.AsSpan() : throw new ArgumentNullException(nameof(bits)))
```

Comment:
I think you can also do: `(Span<int>)(bits ?? throw new ArgumentNullException())` now; not sure which is easier to read though

Reply:
I like yours better.  I'll change it.